src/utils/filterDictionaryByKeys.js

```diff
@@ -0,0 +1,5 @@
+
+export default function filterDictionaryByKeys(dict, keys){
+  const keysList = [].concat(keys)
+  return Object.fromEntries(Object.entries(dict).filter(([k,v]) => keysList.includes(k)));
+}
```

Comment:
This should go according to comment above but could otherwise be expressed as:
```js
import {filter} from 'rambda'

const filterObjectWithKeys = (object, keys) => filter(([_, key]) => keys.includes(key), object)
```